### ISSUE TITLE
How to preserve original file name on build?

### ISSUE THREAD

**Issue by matthewhoth** [gatsbyjs/gatsby#10026]
## Summary
I would like to know how to preserve a file's name after production build. Eg. Prevent [presentation.pdf] from being renamed to [static_1sghgh44fasd3.pdf]

## Relevant information
I am importing the file and then passing it the the html download atttribute. 

`import presentation from '../docs/presentation.pdf'
<a href="" download={presentation}>`

The resulting downloaded pdf name is a very long one that does not match "presentation"
### Environment (if relevant)

Running on the latest version of Gatsby and React.

### File contents (if changed)

`gatsby-config.js`: N/A 
`package.json`: N/A 
`gatsby-node.js`: N/A 
`gatsby-browser.js`: N/A 
`gatsby-ssr.js`: N/A

**Comment by jgierer12** [comment]
You can use an [inline webpack loader](https://webpack.js.org/concepts/loaders/#inline) with custom options:

```js
import presentation from "url-loader?name=[path][name].[ext]!../docs/presentation.pdf";
```

**Comment by matthewhoth** [comment]
> 
> 
> You can use an [inline webpack loader](https://webpack.js.org/concepts/loaders/#inline) with custom options:
> 
> ```js
> import presentation from "url-loader?name=[path][name].[ext]!../docs/presentation.pdf";
> ```

Hey jgierer! Thanks for the reply. I'm not that familar with webpack so I tried using the loader with the exact options you provided. The result was a longer and even more strange name.

After messing around for a little while I found that ``import presentation file-loader?name=[name].[ext]!../docs/presentation.pdf``
results in the file being named "[website url]_presentation.pdf

I feel that I must to missing one little part to get rid of the website name. Any idea what that might be?
